recipe/patches/0001-Darwin-Add-arch-x86_64-to-UnicCCompiler-ld_args.patch

```diff
@@ -1,25 +1,24 @@
-From 0fcc174b9cef0d93ebf1c95fd942c1368132cfdc Mon Sep 17 00:00:00 2001
+From 42c10724b545cb3b0bb2d98f2e85bd1dcac6b80a Mon Sep 17 00:00:00 2001
 From: Ray Donnelly <mingw.android@gmail.com>
 Date: Wed, 16 Aug 2017 08:48:16 +0100
-Subject: [PATCH 02/22] Darwin: Add -arch x86_64 to UnicCCompiler ld_args
+Subject: [PATCH 01/30] Darwin: Add -arch x86_64 to UnicCCompiler ld_args
 
 ---
  Lib/distutils/unixccompiler.py | 1 +
  1 file changed, 1 insertion(+)
 
 diff --git a/Lib/distutils/unixccompiler.py b/Lib/distutils/unixccompiler.py
-index 4d7a6de..3f05cf7 100644
+index 4d7a6de740..3f05cf7f8c 100644
 --- a/Lib/distutils/unixccompiler.py
 +++ b/Lib/distutils/unixccompiler.py
-@@ -200,6 +200,8 @@ class UnixCCompiler(CCompiler):
+@@ -200,6 +200,7 @@ class UnixCCompiler(CCompiler):
  
                  if sys.platform == 'darwin':
                      linker = _osx_support.compiler_fixup(linker, ld_args)
-+                    from distutils.util import get_host_platform
-+                    ld_args = ['-arch', get_host_platform().split("-")[-1]] + ld_args
++                    ld_args = ['-arch', 'x86_64'] + ld_args
```

Comment:
Can you revert this change?

Reply:
Sure. Working on it...
I just wonder why we needed that in the first place, Maybe @mingwandroid can chime-in.

Reply:
To do with fat objects back when the tools supported (and defaulted) to i686 AFAIR, probably fine to jettison at this point.